examples/server_settimestamp.c

```diff
@@ -0,0 +1,94 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+/**
+ * Faking source timestamp
+ * ----------------------------
+ *
+ * This example shows how to write data with a modified source timestamp
+ * When showing the data in UAExpert, the source timestamp will always be half an hour being the server timestamp
```

Comment:
Spelling error?
"be half an hour **being** the server timestamp"
being -> behind

Reply:
fixed.